--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@ cmake_minimum_required(VERSION 2.8.12)
 
 # Standalone build. Find / include everything neccessary.
 set(KF5_VERSION "5.17.0")
-set(GPGMEPP_VERSION "5.1.41")
+set(GPGMEPP_VERSION "1.8.0")
 set(KMIME_VERSION "5.1.40")
 set(LIBKLEO_VERSION "5.1.41")
 set(QT_REQUIRED_VERSION "5.4.0")
@@ -48,7 +48,7 @@ set_package_properties(KF5DocTools PROPE
 # Kdepimlibs packages
 find_package(KF5Libkleo ${LIBKLEO_VERSION} CONFIG REQUIRED)
 find_package(KF5Mime ${KMIME_VERSION} CONFIG REQUIRED)
-find_package(KF5Gpgmepp ${GPGMEPP_VERSION} CONFIG REQUIRED)
+find_package(QGpgme ${GPGMEPP_VERSION} CONFIG REQUIRED)
 
 find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Widgets Test Network)
 
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -63,7 +63,7 @@ ecm_mark_as_test(test_verify)
 target_link_libraries(test_verify
   KF5::Libkleo
   Qt5::Test
-  KF5::QGpgme
+  QGpgme
   KF5::CoreAddons
   KF5::I18n
   Qt5::Widgets
@@ -104,13 +104,13 @@ if(USABLE_ASSUAN_FOUND)
   if(WIN32)
     target_link_libraries(test_uiserver
       ${ASSUAN_VANILLA_LIBRARIES}
-      KF5::QGpgme
+      QGpgme
       ws2_32
     )
   else()
     target_link_libraries(test_uiserver
       ${ASSUAN_PTHREAD_LIBRARIES}
-      KF5::QGpgme
+      QGpgme
   )
   endif()
 
